sql/5_0_2-to-6_0_0_upgrade.sql

```diff
@@ -2440,3 +2440,6 @@ VALUES
 (NULL, "SARS-COV-2 (COVID-19) vaccine, vector non-replicating, recombinant spike protein-ChAdOx1, preservative free, 0.5 mL ", "COVID-19 vaccine, vector-nr, rS-ChAdOx1, PF, 0.5 mL", 210, @codetypeid, '', 0, 0, '', '', '', 1);
 #EndIf
 
+#IfNotColumnType facility weno_id
+ALTER TABLE `facility` ADD `weno_id` VARCHAR(10) NULL DEFAULT NULL
```

Comment:
remove the first NULL and place a semicolon at the end

Reply:
Also, this need to go into the `sql/5_0_2-to-6_0_0_upgrade.sql` file (and remove it from this file). I will add this to the patch sql file when we bring it into the next 5.0.2 patch.

Reply:
updated